Sources/KeyboardShortcuts/KeyboardShortcuts.swift

```diff
@@ -37,6 +37,26 @@ public enum KeyboardShortcuts {
 	*/
 	static var isPaused = false
 
+	public static var isEnabled = true {
+		didSet {
+			guard isEnabled != oldValue else {
+				return
+			}
+
+			CarbonKeyboardShortcuts.setUpRelevantEventHandler()
+		}
+	}
+
+	public static var isMenuOpen = false {
```

Comment:
Would you be able to add doc comments to these properties?

You can use https://chat.openai.com to help out with the text.